pandas/core/frame.py

```diff
@@ -924,21 +924,26 @@ def from_dict(cls, data, orient='columns', dtype=None, columns=None):
         return cls(data, index=index, columns=columns, dtype=dtype)
 
     def to_dict(self, orient='dict', into=dict):
-        """Convert DataFrame to dictionary.
+        """
+        Convert DataFrame to dictionary.
+
+        Method converting the DataFrame to a Python dictionary,
+        the type of the key-value pairs can be customized with
+        the parameters (see below).
```

Comment:
A bit repetitive, I wouldn't repeat that it converts to a dictionary again, and directly explain that you can customize.